tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -224,10 +224,7 @@ private static Date getDate(int year, int month, int date, int hour, int minute,
 
     public void setCachingEnabled(boolean enabled) {
         exCache.setCachingEnabled(enabled);
-    }
-
-    public void setCacheOnly(boolean cacheOnly) {
-        exCache.setCacheOnly(cacheOnly);
+        icuServiceBuilder.setCachingEnabled(enabled);
```

Comment:
This is the most essential change, used with enabled = false only by GenerateExampleDependencies, not while ST is running